platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -54,7 +54,13 @@ class ConfigurationCacheProblemsSummary(
     var overflowed: Boolean = false
 
     private
-    var totalProblemCount: Int = 0
+    var interrupted: Boolean = false
+
+    private
+    val totalProblemCount: Int
+        get() = deferredProblemCount +
+            suppressedProblemCount +
+            if (interrupted) 1 else 0
```

Comment:
❓  Why do we make an assumption that there is only one interrupting problem?

Also, this change seems unrelated to the PR.